src/rdo.rs

```diff
@@ -190,12 +190,8 @@ pub fn rdo_mode_decision(
   let partition_start_x = (bo.x & LOCAL_BLOCK_MASK) >> xdec << MI_SIZE_LOG2;
   let partition_start_y = (bo.y & LOCAL_BLOCK_MASK) >> ydec << MI_SIZE_LOG2;
 
-  for &skip in &[false, true] {
-    // Don't test skipped blocks at higher speed levels
-    if fi.config.speed > 1 && skip {
-      continue;
-    }
-
+  let skip = false;
+  {
```

Comment:
Considering this loop will be permanently removed (#373) I recommend removing the enclosing scope (braces) as well.

Reply:
:) I initially removed the { .. } then git diff showed me lots of changes, so left the { }? :)

Reply:
So, we prefer remove { }, right?

Reply:
Yes, the whole enclosing scope will show up in a `git diff` if the indentation changes as it will when the outer scope is removed, but that is fine.